src/lib.rs

```diff
@@ -11,9 +11,11 @@ use crate::renderer::{Renderer, RendererOptions, SceneRendererGeometry};
 use crate::ui::Ui;
 
 pub mod importer;
+pub mod logger;
 
 mod camera;
 mod convert;
+mod fs;
```

Comment:
Maybe name this `platform`? I imagine there will be more than fs functions in there pretty soon.